src/oph/heratepalvelu/tep/kestojenUudelleenlaskentaHandler.clj

```diff
@@ -42,7 +42,7 @@
               kestot            (nh/oppijan-jaksojen-kestot
                                   concurrent-jaksot opiskeluoikeudet)
               jakso-key         (nh/ids jakso)]
-          (if-let [new-kesto (get kestot jakso-key)]
+          (if-let [new-kesto (get kestot jakso-key 0)]
```

Comment:
Tää on sikäli bugialtista, että olis kuviteltavissa, että kestot-mapista puuttuu kyseinen avain jostain muusta syystä kuin siksi, että kesto yritettiin laskea mutta epäonnistuttiin.  Mun puolesta näin voi tehdä, mutta enemmän eksplisiittisiä mahdollisuuksia olis esim. että `oppijan-jaksojen-kestot` laittais kestot-mappiin jonku `:jaksoa-ei-loydy`-keywordin numeron sijaan silloin ku laskennassa on joku ongelma.

Vakavuus: ehdotus